grpc-client-rx-utils/build.gradle.kts

```diff
@@ -7,14 +7,9 @@ plugins {
 
 dependencies {
   api("io.reactivex.rxjava3:rxjava:3.0.6")
-  api("io.grpc:grpc-stub:1.33.1")
+  api("io.grpc:grpc-stub:1.35.0")
```

Comment:
(I also took the opportunity to bump the grpc version and remove the guava constraint which is no longer needed as grpc has bumped their dependency version to 30.0-android)